src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -761,8 +763,9 @@ trait BronzeTransforms extends SparkSessionWrapper {
           } else col("Timestamp")
 
           baseDF
-            .withColumn("Timestamp", fixDupTimestamps)
+            .withColumn("DerivedTimestamp", fixDupTimestamps)
```

Comment:
This is a great idea. I'm thinking we might should do something similar for all the complex field derivations. The stageId derivation could be getting null for a similar reason.

What I mean by something similar is give them a temporary column name and then rename them, drop the originals and rename it back to the target name.

* executorIdOverride
* progressCol
* stageIDColumnOverride
* stageAttemptIDColumnOverride